budget-app/src/main/webapp/recordSpending.html

```diff
@@ -62,4 +63,71 @@ <h1>Record your spending</h1>
     </div>
     <script src="recordSpending.js"></script>
   </body>
+=======
```

Comment:
This is the separator between your code and the existing code. It should also be removed.